### PR TITLE
[Ready] Recipe for Crackers

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Recipe for Crackers"

#### Purpose of change
Now you can make crackers, for your cracker needs.  Like SuperTurkey dressing for POSTAPOC Thanksgiving. 

#### Describe the solution
Adds the recipe for Crackers based on this recipe  https://www.thepioneerwoman.com/food-cooking/recipes/a98773/how-to-make-crackers/

#### Describe alternatives you've considered
Selling real estate.  You need to imagine this place without the tear in reality spewing forth heinous screams and pollutants from a dimension of endless despair and agony.

#### Testing
TBD

### CODE REVIEW

**Comment by Maleclypse** [comment]
note to self.  rebase looks like other recipes in this file have been updated since I last updated master.

**Comment by ThomasLinkin** [comment]
can i make Cracker & Cheese ?

**Comment by Maleclypse** [comment]
> can i make Cracker & Cheese ?

Cracker and Cheese already exists in 0.F-dev.  I did that a few weeks back.

**Comment by Maleclypse** [comment]
Ooof I only wanted to keep changes in wheat.json and data/json/recipes/food/bread.json.

File: data/json/recipes/recipe_obsolete.json [hunk 1]
```diff
@@ -2775,7 +2775,7 @@
   },
   {
     "type": "recipe",
-    "result": "rifle_223",
+    "result": "bodypillow",
```
Comment (Maleclypse) [inline data/json/recipes/recipe_obsolete.json:2778]:
```suggestion
    "result": "rifle_223",
```

**Pull request merged**